pages/formpage.js

```diff
@@ -0,0 +1,9 @@
+import Form from "@/components/Form";
+
+export default function FormPage() {
+  return (
+    <>
+      <Form />
+    </>
+  );
```

Comment:
If there is just the `Form` to render, you don't need the `fragments `and the round brackets

Reply:
Ok, but now i have this error.

<img width="461" alt="Bildschirmfoto 2023-07-13 um 16 39 58" src="https://github.com/SelcoD/CampingFriends/assets/132760864/865d07dc-9b9a-46de-aeb2-f6814d9c139c">

Reply:
i dont have this error anymore. i fixed it. thanks